test/fixtures/preset-options/use-builtins-all-exec/exec.js

```diff
@@ -0,0 +1 @@
+import "babel-polyfill";
```

Comment:
Because this is an exec.js test, it will actually run all the requires so it would fail if it wasn't found?